lib/cisco_node_utils/cmd_ref/acl.yaml

```diff
@@ -0,0 +1,33 @@
+# Command Reference Common ACL
+#
+# For documentation please see:
+#   - README_YAML.md
+#
+---
+_template:
+  config_get: 'show run aclmgr'
```

Comment:
Hi Mike
show run aclmgr all will display copp acl as well (which is created by 3rd party process and owned by them)
show run aclmgr will display acls that owned by aclmgr only